ci/conda/recipes/morpheus-libs/morpheus_llm_test.sh

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# llm library and tests are dependent on a number of pypi packages - fixme
+pip install --no-input milvus==2.3.5
```

Comment:
My concern here is that these could potentially go out of sync with those in `dependencies.yaml`. 

It should be possible to define a requirements output in the `dependencies.yaml` file something like:
```yaml
  examples-llms:
    output: requirements
    matrix:
      cuda: ["12.1"]
      arch: [x86_64]
    includes:
      - example-llms-pip

  example-llms-pip:
    common:
      - output_types: [requirements]
        packages:
          - langchain==0.1.16
          - langchain-nvidia-ai-endpoints==0.0.11
          - faiss-gpu==1.7.*
          - google-search-results==2.4
          - nemollm==0.3.5
```

Then do:
```shell
rapids-dependency-file-generator --output requirements --file-key examples-llms --matrix "cuda=12.1;arch=x86_64;py=3.10" > requirements.txt
```

Reply:
I agree, entirely. I was planning on fixing that up exactly as you have noted; but slipped thru the cracks.  Experimenting/testing conda validation is a time-consuming pita :(